client/components/badge/style.scss

```diff
@@ -0,0 +1,22 @@
+.badge {
+	display: inline-block;
+	border-radius: 100px;
+	padding: 2px 11px;
+	text-align: center;
+	font-size: 14px;
+	line-height: 18px;
+}
+
+.badge--warning {
+	color: #535d56;
```

Comment:
We can't use hex values for color. Can you make a new PR and replace these hex values with sass variables from the _colors file?